### DISCUSSION TITLE
how to enable amd gpu for ollama ?

### DISCUSSION THREAD

**Discussion by hemangjoshi37a** [open-webui/open-webui#484] [General]
how to enable amd gpu for ollama ?

**Comment by justinh-rahb** [comment]
Hello @hemangjoshi37a,

For assistance with enabling an AMD GPU for Ollama, I would recommend reaching out to the Ollama project support team or consulting their official documentation. Ollama WebUI is a separate project and has no influence on whether or not your AMD GPU is used by Ollama.
